### PR TITLE
Refactor iter4 sa

### PR DESCRIPTION
**Pull request by StevieAmb**
## Branch title

- refactor-iter4-SA

## Changes made

- Added the saveStorage method, worked on localStorage and stringify methods, and updated some CSS.

## Anything Refactored

- The CSS.

## Anything needed

- Continued work on JSON and localStorage.

## Contributors

* Stevie Ambroise
* Alex Fritz
* Devon Wigle

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -157,7 +157,7 @@ main {
   justify-content: center;
   align-items: center;
   background-color: #FFF;
-  border: none;
+  border: 0;
```
Comment (alexmfritz) [inline styles.css:160]: Thanks for swapping these to fit Turing's style guide!

File: src/main.js [hunk 2]
```diff
@@ -273,9 +172,110 @@ function changeCircleColor(element) {
     hide(element, 'purple');
   } else if (currentActivity.category === 'Exercise') {
     hide(element, 'red');
-  }
+  };
 };
 
+function highlightStudy() {
```
Comment (alexmfritz) [inline src/main.js:186]: Spooky rearranging everything 😮

File: src/main.js [hunk 3]
```diff
@@ -74,48 +74,10 @@ categoryBox.addEventListener('click', function (event) {
 });
 
 //functions
-function displayHomePage() {
-  clearInput();
-  clearButton();
-  clearErrorMessage();
-  clearCircle();
-  hide(createNewActivityView, 'hidden');
-  show(userActivityInputView, 'hidden');
```
Comment (devonwigle) [inline src/main.js:83]: I think the rearranging here will make things easier to find in the future.

File: src/main.js [hunk 4]
```diff
@@ -237,7 +145,7 @@ function showErrorMessages() {
 function validateCategory() {
   if (selectedCategory === '') {
     show(categoryError, 'visibility');
-  }
+  };
```
Comment (devonwigle) [inline src/main.js:148]: The semi-colons!

**Review by devonwigle — APPROVED** [review]

**Pull request merged**